Date/Date64Module.pbi

```diff
@@ -0,0 +1,741 @@
+﻿;   Description: Support for enlarged date range (64 bit unix timestamp)
```

Comment:
The "official" version of the module already exists in the archive. Your version only uses a different programming style and removes some informative comments. Please accept programming styles from others and don't be angry that I won't include this module.